addons/resource.language.en_gb/resources/strings.po

```diff
@@ -23441,3 +23447,9 @@ msgstr ""
 msgctxt "#39193"
 msgid "If music is being played, the screensaver will never be activated"
 msgstr ""
+
+#. Help text of setting "Allow use DXVA Video Super Resolution" with label #13417
+#: system/settings/settings.xml
+msgctxt "#39194"
+msgid "Enables advanced DXVA upscaler using NVIDIA \"RTX Video Super Resolution\" or \"Intel Video Super Resolution\".[CR]Requires render method set to Auto or DXVA.[CR]Used when video source is 1080p or less (progressive only) and source resolution is lower than display resolution.[CR]It's only available on specific hardware: NVIDIA RTX 40x, RTX 30x and Intel Arc A770, A750."
```

Comment:
Minor, wondering if `[CR]Requires render method set to Auto or DXVA.[CR]` is actually needed on the help string since you already only show the setting if those methods match on `videoplayer.rendermethod`. Would make the string smaller

Reply:
This was thinking that someone who has Pixel shaders set up might not know how to turn on Super Resolution but the help text isn't actually displayed either so it's probably better to remove it.

![screenshot00011](https://github.com/xbmc/xbmc/assets/58434170/9a8c0980-9f64-4370-8d69-2b015cd9ec4b)